code/game/machinery/harvester.dm

```diff
@@ -0,0 +1,174 @@
+/obj/machinery/harvester
+	name = "organ harvester"
+	desc = "An advanced machine used for harvesting organs and limbs from the deceased."
+	density = TRUE
+	icon = 'icons/obj/machines/harvester.dmi'
+	icon_state = "harvester"
+	verb_say = "states"
+	state_open = FALSE
+	idle_power_usage = 50
+	circuit = /obj/item/circuitboard/machine/harvester
+	light_color = LIGHT_COLOR_BLUE
+	var/interval = 20
+	var/harvesting = FALSE
+	var/list/operation_order = list() //Order of wich we harvest limbs.
+	var/allow_clothing = FALSE
+	var/allow_living = FALSE
+
+/obj/machinery/harvester/Initialize()
+	. = ..()
+	if(prob(5))
+		name = "auto-autopsy"
```

Comment:
?

Reply:
some guy suggested it, it's a fun little reference

Reply:
I think in the long run `prob(5)` is too common, I'd suggest using `prob(1)`, but it's up to you, of course.